### PR TITLE
Fix options configuration for latest version of selenium-standalone

### PR DESCRIPTION
**Pull request by JSMike**

### CODE REVIEW

**Comment by JSMike** [comment]
Also adds option to add stopOnExit config to the grunt-selenium-standalone task, this will kill the selenium process if grunt fails before you stop selenium.

Example config:

```
{
    selenium_standalone: {
        server: {
            stopOnExit: true,
            seleniumVersion: '2.47.0',
            seleniumDownloadURL: 'http://selenium-release.storage.googleapis.com',
            drivers: {
                chrome: {
                    version: '2.20',
                    arch: process.arch,
                    baseURL: 'http://chromedriver.storage.googleapis.com'
                }
            }
        }
    }
}
```

To use this version now, while the repo owner is MIA, update your package.json entry for grunt-selenium-standalone to match:

```
    "grunt-selenium-standalone": "git+https://github.com/JSMike/grunt-selenium-standalone.git#v0.1.5"
```

**Comment by JSMike** [comment]
@zs-zs I see you were active again last week. Could you please look at this PR?

**Comment by JamesMGreene** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-selenium-standalone",
   "description": "Grunt tasks for running a standalone Selenium server using the popular selenium-standalone package",
-  "version": "0.1.2",
+  "version": "0.1.5",
```
Comment (JamesMGreene) [inline package.json:4]: This should be `0.2.0` since you added a new feature (`stopOnExit`).

**Comment by JamesMGreene** [comment]
@JSMike: Heads up: I'm going to start picking your `master` branch apart and merging/editing each of the changes you've generously offered up but rest assured that it will all make its way into the mainline now. 💝

**Comment by JamesMGreene** [comment]
@JSMike I already merged your first 2 commits with corrected formatting but I'd like to propose something different for your 3rd commit. I'll submit a PR tonight.

**Comment by JamesMGreene** [comment]
@JSMike Your third commit was addressed via 290f6e0ca58583bcb3c21715f5e0d22699972eac

Updated, versioned, released, and published as `v1.0.0`.

**Comment by JamesMGreene** [comment]
Sorry, had to release a quick patch fix: `v1.0.1`  :confused:

**Comment by JSMike** [comment]
Thank you, I was overly busy this weekend. If you want me to review anything just tag me

**Pull request closed without merging**